specs/language/overloading.tex

```diff
@@ -324,12 +324,18 @@
 \p Otherwise, the rank of a conversion sequence is determined by considering the
 rank of each conversion.
 
-\p Conversion sequence ranks are ordered such that \textbf{Exact
-Match} rank is better than \textbf{Exact Match Extension} rank, which is better
-than \textbf{Promotion} rank, which is better than \textbf{Extension} rank,
-which is better than \textbf{Conversion} rank, which is better than
-\textbf{Truncation} rank. The rank of a conversion sequence is the rank of the
-worst ranked conversion in the sequence.
+\p Conversion sequence ranks are ordered from better to worse as:
+\begin{itemize}
```

Comment:
Does "itemize" include a numbered list? Not a big deal either way, just curious. This is WAY more readable, though!

Reply:
The current formatting for `itemize` is:
<img width="640" alt="Screenshot 2024-09-04 at 7 01 15 PM" src="https://github.com/user-attachments/assets/7c5dac0d-58c8-4d68-9fcf-c597c15d3f21">

I could instead use an `enumerate` block if you think numbering would be more clear.

Reply:
Really minor, but `enumerate` might be better, especially when you're dealing with an ordered list to begin with.